guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,325 @@
+
+A [Lightning address](https://lightningaddress.com/) LWS service gives users a human-readable address that can be re-used to receive lightning payments. An example is bosch@bitcoin.design.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Lightning-address.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile@2x.png"
+   alt-text = "Examples of lightning addresses"
+   width = 800
+   height = 348
+%}
+
+### Receive payments offline
+
+Non-custodial lightning wallets need to be online to receive payments. [Daily spending wallets]({{ '/guide/daily-spending-wallet/' | relative_url }}), in particular, are regularly offline, so receiving payments can be difficult.
+
+LWSs can hold onto a payment (without assuming custody) and forward it to the user once they come online. A product that offers this for users is [Greenlight](https://blockstream.com/lightning/greenlight/).
+
+This service is usually required to be offered alongside an LSP. Read more about this [here](https://medium.com/breez-technology/get-ready-for-a-fresh-breez-multiple-apps-one-node-optimal-ux-519c4daf2536).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Offline-payment.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile@2x.png"
+   alt-text = "How an LWS helps users accept payments whilst offline"
+   width = 800
+   height = 348
+%}
+
+### Swaps
+
+Moving bitcoin between lightning and on-chain can be costly and complicated if done by opening and closing payment channels.
+
+LWSs can offer [submarine swaps](https://blog.muun.com/a-closer-look-at-submarine-swaps-in-the-lightning-network/) to allow users to easily move bitcoin between lightning and on-chain without having to manage and deal with payment channels.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps-mobile@2x.png"
+   alt-text = "How an LWS can conduct swaps moving bitcoin between on-chain and lightning"
+   width = 800
+   height = 348
+%}
```

Comment:
Need some capital Bs in these images because every other title sentence fragment is capitalized at the beginning.

- ~"bitcoin on lightning"~ -> "Bitcoin on lightning"
- ~"bitcoin on-chain"~ => "Bitcoin on-chain"